### PR TITLE
Leash effect examples/description improved

### PR DESCRIPTION
**Pull request by Asleeepp**
### Description

Updates to the Leash effect's description and examples.

Probably one of the more basic PR's you guys have gotten but uh... there was an issue for it.
 
https://github.com/SkriptLang/Skript/issues/6234

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/effects/EffLeash.java [hunk 1]
```diff
@@ -34,8 +34,11 @@
 import ch.njol.util.Kleenean;
 
 @Name("Leash entities")
-@Description("Leash living entities to other entities.")
-@Examples("leash the player to the target entity")
+@Description("Leash living entities to other entities. When trying to leash an Ender Dragon, Wither, Player, or a Bat, this effect will not work. See: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/LivingEntity.html#setLeashHolder(org.bukkit.entity.Entity) ")
+@Examples({
+			"on right click:",
+			"\tleash event-entity to player",
+			"\tsend \"&aYou leashed %2%event-entity%!\" to player"})
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffLeash.java:41]:
```suggestion
				"\tleash event-entity to player",
				"\tsend \"&aYou leashed &2%event-entity%!\" to player"})
```
We indent when the skript code should be indented for readability. :)

File: src/main/java/ch/njol/skript/effects/EffLeash.java [hunk 2]
```diff
@@ -34,8 +34,11 @@
 import ch.njol.util.Kleenean;
 
 @Name("Leash entities")
-@Description("Leash living entities to other entities.")
-@Examples("leash the player to the target entity")
+@Description("Leash living entities to other entities. When trying to leash an Ender Dragon, Wither, Player, or a Bat, this effect will not work. See: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/LivingEntity.html#setLeashHolder(org.bukkit.entity.Entity) ")
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffLeash.java:37]:
```suggestion
@Description("Leash living entities to other entities. When trying to leash an Ender Dragon, Wither, Player, or a Bat, this effect will not work.")
```

**Review by sovdeeth — CHANGES_REQUESTED** [review]
Thanks for the PR

File: src/main/java/ch/njol/skript/effects/EffLeash.java (hunk 1 above)
Comment (AyhamAl-Ali) [reply src/main/java/ch/njol/skript/effects/EffLeash.java:41]:
```suggestion
				"\tleash event-entity to player",
				"\tsend \"&aYou leashed &2%event-entity%!\" to player"
})
```
We indent when the skript code should be indented for readability. :)

**Review by AyhamAl-Ali — APPROVED** [review]
Other than Sovde's suggestions, LGTM.

**Comment by Asleeepp** [comment]
gonna open a new pr bc i merged from master on accident 🥴

**Pull request closed without merging**